lib/infrastructure/crowdaction/crowdaction_comment_dto.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+part 'crowdaction_comment_dto.freezed.dart';
+part 'crowdaction_comment_dto.g.dart';
+
+@freezed
+class CrowdactionCommentDto with _$CrowdactionCommentDto {
+  /// {@macro crowdaction_comment_dto}
```

Comment:
Does this macro key exist somewhere? And if not, we don't need it

Reply:
It's autogenerated, i can clear the comments https://brickhub.dev/bricks/model/0.5.1